src/main/java/net/danygames2014/unitweaks/UniTweaks.java

```diff
@@ -2,13 +2,11 @@
 
 import net.danygames2014.unitweaks.util.Config;
 import net.glasslauncher.mods.gcapi3.api.ConfigRoot;
-import net.modificationstation.stationapi.api.mod.entrypoint.Entrypoint;
-import net.modificationstation.stationapi.api.util.Null;
+import net.modificationstation.stationapi.api.util.Namespace;
 import org.apache.logging.log4j.Logger;
 
 public class UniTweaks {
-    @Entrypoint.Logger
-    public static final Logger logger = Null.get();
+    public static final Logger logger = Namespace.resolve().getLogger();
```

Comment:
```suggestion
    public static Logger logger = Null.get();
```
The fix you are using will only work for StationAPI alpha.4. To let the mod still be backwards compatible instead the word final can be removed

Reply:
Yes, that's true. I have reverted that - thanks for the heads-up!